gravitee-am-ui/src/app/domain/applications/application/advanced/factors/mfa-activate/mfa-activate.component.ts

```diff
@@ -22,41 +23,52 @@ import {Component, EventEmitter, Input, OnInit, Output} from "@angular/core";
 })
 export class MfaActivateComponent implements OnInit {
 
+  constructor(private organizationService: OrganizationService) {
+  }
+
   @Input() enrollment: any;
   @Input() adaptiveMfaRule: string;
   @Input() riskAssessment: any;
-  @Output("settings-change") settingsChangeEmitter: EventEmitter<any> = new EventEmitter<any>();
-
+  @Output('settings-change') settingsChangeEmitter: EventEmitter<any> = new EventEmitter<any>();
   currentMode: any;
+  factors: any[];
 
   private static modeOptions = {
     OPTIONAL: {
-      "label": "Optional",
-      "message": "Choose the period of time users can skip MFA. Default time is 10 hours."
+      'label': 'Optional',
+      'message': 'Choose the period of time users can skip MFA. Default time is 10 hours.',
+      'ee': false,
     },
     REQUIRED: {
-      "label": "Required",
-      "message": "MFA will always be displayed."
+      'label': 'Required',
+      'message': 'MFA will always be displayed.',
+      'ee': false,
     },
     CONDITIONAL: {
-      "label": "Conditional",
-      "message": "Set conditions that will display MFA based on the user’s information",
-      "warning": "You need to install the <b> GeoIP service </b> plugin to use the geoip based variables"
+      'label': 'Conditional',
+      'message': 'Set conditions that will display MFA based on the user’s information',
+      'warning': 'You need to install the <b> GeoIP service </b> plugin to use the geoip based variables',
+      'ee': false,
     },
     INTELLIGENT: {
-      "label": "Risk-based",
-      "message": "Configure the thresholds that will display MFA based on risks.",
-      "warning": "You need to install the <b> GeoIP service </b> and <b> Risk Assessment </b> plugins to use Risk-based MFA"
+      'label': 'Risk-based',
+      'message': 'Configure the thresholds that will display MFA based on risks.',
+      'warning': 'You need to install the <b> GeoIP service </b> and <b> Risk Assessment </b> plugins to use Risk-based MFA',
+      'ee': true,
     },
   };
 
   private static modes = Object.keys(MfaActivateComponent.modeOptions)
     .map(key => MfaActivateComponent.modeOptions[key])
 
   ngOnInit(): void {
+    this.organizationService.factors().subscribe(data => {
```

Comment:
I'm not sure if risk-management plugin comes from the this.organizationService.factors() or other endpoints exist for plugins in the AM

Reply:
So which endpoint gives back the information about this plugin?
I see we have factors, identities, resources, certificates ...

Reply:
It's fairly simple to implement the License endpoint and check from there once you have `gravitee-node` up to date e.g. in APIM

```java
@Path("/license")
public class GraviteeLicenseResource extends AbstractResource {

    @Inject
    private NodeLicenseService nodeLicenseService; // from gravitee-node

    @GET
    @Produces(MediaType.APPLICATION_JSON)
    @Consumes(MediaType.APPLICATION_JSON)
    public GraviteeLicense get() {
        return GraviteeLicenseMapper.INSTANCE.map(
            GraviteeLicenseEntity
                .builder()
                .tier(nodeLicenseService.getTier())
                .packs(nodeLicenseService.getPacks())
                .features(nodeLicenseService.getFeatures())
                .build()
        );
    }
}
```

Reply:
TBD in alpha.2